specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/application/rfc7230/validation/valid/client.rpt

```diff
@@ -39,9 +38,11 @@ read zilla:begin.ext ${http:beginEx()
 
 read "response1"
 read closed
-read notify RESPONSE_ONE_RECEIVED
+read notify RESPONSE_1_RECEIVED
```

Comment:
I think we use words for barriers everywhere by convention, so let's keep the same here.

Reply:
done